test/odbc/config.txt

```diff
@@ -1,6 +1,15 @@
-ODBC_DRIVER_NAME=
-BABEL_DB_SERVER=
-BABEL_DB_PORT=
-BABEL_DB_USER=
-BABEL_DB_PASSWORD=
-BABEL_DB_NAME=master
\ No newline at end of file
+MSSQL_ODBC_DRIVER_NAME=ODBC_Driver_17
```

Comment:
As we discussed in the call, we probably can dispense with the the prefixes MSSQL_ and PSQL_, and possibly more if we add drivers.
Currently, the prefixes are used to distinguish between drivers, or driver types. 
For this purpose we could add another flag, e.g. DRIVER_TYPE which would specify which driver this is.
The flag could be set like this in this file:
DRIVER_TYPE=MSSQL or
DRIVER_TYPE=PSQL

More drivers could be added in similar fashion. 

Having said that, this is still an early thought. If there are reasons that we need to keep the prefixes then we should.

Reply:
It looks like OdbcHandler::ParseConfigFile() depends on these names being unique.
Simply removing the prefixes would break ParseConfigFile().

That is not to say that the parsing could not be changed. However, currently there are other dependencies on the specific names, e.g. the expected environment variable names as in OdbcHandler::SetConnectionString ().

Currently a simple map stores each value. The change would require a list of structures, or a list\vector of maps, where each item on the list would be a map with the flag names. Just brainstorming, did not think through what the best structure would be (if we do want to change it)